sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -13,9 +13,12 @@
 
 - `NamespaceProperties` interface property "messageSku" type changed from "string" to string literal type "Basic" | "Premium" | "Standard". [PR 11810](https://github.com/Azure/azure-sdk-for-js/pull/11810)
 
+- Internal improvement - Previously, each `RequestResponseLink.sendRequest` call adds an "onMessage" listener to the `ReceiverEvents.message` event and keeps discarding the responses that did not match the request-id and returns the response if matched. This has been improved to reuse the listener for all the requests by maintaining a map of deferred promises that would be resolved(or rejected) upon receiving a message event.
```

Comment:
Shouldnt this PR be having a changelog entry for core-amqp instead of service bus?

Reply:
Oops!

Reply:
Moved